app/views/course/achievement/achievements/_achievement.html.slim

```diff
@@ -24,9 +24,13 @@
 
   td
     div.btn-group
-      - if can?(:manage, achievement)
+      - if can?(:award, achievement)
```

Comment:
Okay, but I think `#count` will trigger extra queries, can you check ? 

And `#length` will fix that.